pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter_test.go

```diff
@@ -2560,8 +2584,9 @@ var _ = Describe("Function getNumberOfPodInterfaces()", func() {
 				Pod:    &v1.PodNetwork{},
 				Multus: &v1.MultusNetwork{NetworkName: "testnet1"},
 			},
+			Name: "testnet",
 		}
-		iface1 := v1.Interface{Name: net1.Name}
+		iface1 := v1.Interface{Name: "testnet"}
```

Comment:
How is this related to the PR?

Reply:
(If it is an unrelated fix, I don't mind piggy backing it in this PR, but let's at least give it a dedicated commit)

Reply:
`^[A-Za-z0-9-_]+$` doesn't allow empty string as interface name

Reply:
see my comment above ^^

Reply:
Nevermind. Got it now

Reply:
Nit: Now that `net1` has a Name, can't we revert to `net1.Name`?

Reply:
@jean-edouard Yes, you have a point. Done.